library/stat.h

```diff
@@ -124,6 +124,31 @@ static const struct eblob_stat_entry eblob_stat_default_global[] = {
 		EBLOB_GST_DATASORT_SINGLE_PASS_VIEW_USED_NUMBER,
 		{0}
 	},
+	{
+		"datasort_number_touched_blobs",
```

Comment:
here and after - `datasort_touched_blobs_number`?

Reply:
done